crates/yakui-app/src/lib.rs

```diff
@@ -198,25 +193,11 @@ impl Graphics {
         }
 
         match event {
-            Event::WindowEvent {
-                event: WindowEvent::CloseRequested,
-                ..
-            } => {
-                elwt.exit();
+            WindowEvent::CloseRequested => {
+                event_loop.exit();
             }
 
-            Event::NewEvents(cause) => {
-                if *cause == StartCause::Init {
-                    self.is_init = true;
-                } else {
-                    self.is_init = false;
-                }
```

Comment:
Was this bug fixed in winit? We should fully remove it if so, or keep it in place if this is still a problem.

Reply:
no idea what the bug is but it didn't seem to have an effect?

Reply:
reverted (we think)